Centaurus.Test.DynamicSerializers/TestDynamicSerializersInitializer.cs

```diff
@@ -0,0 +1,22 @@
+﻿using Centaurus.Test.Contracts;
+using Centaurus.Xdr;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Centaurus.Test
+{
+    public static class TestDynamicSerializersInitializer
+    {
+        public static void Init()
+        {
+            var _ = new PrimitiveTypes(); //force loading assembly
+            _.Bool = true;
```

Comment:
Why do we need this line

Reply:
Removed. It turned out that this assembly force-loading code is not needed anymore.